plugins/woocommerce-blocks/assets/js/blocks/product-gallery/inner-blocks/product-gallery-large-image-next-previous/block.json

```diff
@@ -10,23 +10,10 @@
 		"WooCommerce"
 	],
 	"usesContext": [
-		"nextPreviousButtonsPosition",
 		"productGalleryClientId",
 		"postId"
 	],
 	"textdomain": "woocommerce",
-	"supports": {
-		"layout": {
-			"default": {
-				"type": "flex",
-				"verticalAlignment": "bottom"
-			},
-			"allowVerticalAlignment": true,
-			"allowJustification": false,
-			"allowOrientation": false,
-			"allowInheriting": false
-		}
-	},
```

Comment:
I think the alignment option was handful. Especially, that it's native feature. What do you think?

In addition, I think we could add a colors support at least for button's background and potentially for text (if these SVGs react to text color `<`, `>`).

Reply:
https://github.com/woocommerce/woocommerce/pull/54561/commits/7fcba29f764c205793fb229b440a071147f4a5b8

Reply:
>if these SVGs react to text color <, >

I thought so too but theyre not supported